### PR TITLE
Add section_103_message column to institutions

### PR DESCRIPTION
**Pull request by devin-mccurdy**
## Description
Add `section_103_message` column to `institutions`

DB migration PR for https://github.com/department-of-veterans-affairs/gibct-data-service/pull/643

[ZenHub issue](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/7843)

## Testing done
Tested locally and QA reviewed

## Screenshots
N/A

## Acceptance criteria
- [x] Column  `section_103_message` column added to `institutions` table

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by philipwritescode — APPROVED** [review]
~LGTM~
Added a comment below based on the other PR

File: db/migrate/20200414145010_add_section_103_message_column_to_institutions.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddSection103MessageColumnToInstitutions < ActiveRecord::Migration[5.2]
+  def change
+    add_column :institutions, :section_103_message, :string
```
Comment (erluti) [inline db/migrate/20200414145010_add_section_103_message_column_to_institutions.rb:3]: Does it make sense for this to have `default: 'No information available at this time'`?

**Review by philipwritescode — CHANGES_REQUESTED** [review]

File: db/migrate/20200414145010_add_section_103_message_column_to_institutions.rb (hunk 1 above)
Comment (erluti) [reply db/migrate/20200414145010_add_section_103_message_column_to_institutions.rb:3]: (same in the other migration)

Comment (erluti) [reply db/migrate/20200414145010_add_section_103_message_column_to_institutions.rb:3]: If so, you could remove one of the updates in the  `self.add_sec103(version_id)` method

Comment (dmccurdybah) [reply db/migrate/20200414145010_add_section_103_message_column_to_institutions.rb:3]: Definitely, sounds good!

**Review by philipwritescode — APPROVED** [review]

**Review by delli-gatti-michael — APPROVED** [review]

**Pull request merged**